CHANGELOG.md

```diff
@@ -88,8 +88,13 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
 
 #### Alphabet
 
-* Removed seqan3::char_is_valid_for requirement from seqan3::writable_alphabet and seqan3::writable_constexpr_alphabet
+* Removed seqan3::char_is_valid_for requirement from seqan3::writable_alphabet and
```

Comment:
This is still a change due to the new line after and. Maybe change that back? :)

Reply:
I added `detail::` and needed to wrap the line, as it exceeded our line limit :)